### DISCUSSION TITLE
Invalid Master Password on Unlock [SOLVED]

### DISCUSSION THREAD

**Discussion by SocOrg** [dani-garcia/vaultwarden#1379] [Q&A]
Hi bitwarden_rs experts - I have had no issues with running selfhosted bitwarden_rs on a Synology NAS (using it for over a year - thanks to the developers!). For the last couple of days I am seeing an error on iOS devices (both on iphone and ipad) where unlock in APP does not work - have to log out and login again. I am using the bitwardenrs/server:latest image. 

Config and environment setting: https://imgur.com/5bAu4rs

Thanks and sorry about the newbie question.

**Comment by BlackDex** [comment]
What version are you running. I think it's not one of the latest versions available.

**Reply by SocOrg** [reply]
On the admin page I see: Version: 1.10.0-e7b8602e
![image](https://user-images.githubusercontent.com/35916595/107608349-099c2e00-6c3c-11eb-81cc-0667459621e5.png)

**Reply by BlackDex** [reply]
Well that is a very old version from August 2019.
I suggest to create a backup and upgrade to the new version v1.19.0

**Reply by SocOrg** [reply]
Thank you! I have a Synology NAS and I did an update to the image now. Is there a step-by-step backup and upgrade procedure? 

edit: If I just backup the contents of folder /docker/bw-data (as below), can I just overwrite the content of the new installation with the db.sqlite3 file?

![image](https://user-images.githubusercontent.com/35916595/107610420-d9578e00-6c41-11eb-8876-9897d63c2841.png)

**Reply by SocOrg** [reply]
I just reinstalled with the db.sqlite3 file and everything is up and running. Thanks!

Is there a recommended way to back up the database (automatic backup)?

![image](https://user-images.githubusercontent.com/35916595/107618153-6524e680-6c51-11eb-83e3-d25893b8084b.png)

**Reply by BlackDex** [reply]
Great :).
See: https://github.com/dani-garcia/bitwarden_rs/wiki/Backing-up-your-vault
And of course all the other wiki articles.
